index.html

```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+        </div>
+        <div class="art-container">
+          <h2 class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </h2>
+          <p class="profesional-Art-content">
+            A daily
+            selection of privately personalized reads;
+            no accounts or sign-ups required. has been the
+            industry's standard
+          </p>
+          <ul class="profesional-Art-ul">
+            <li class="profesional-Art-li">html</li>
+            <li class="profesional-Art-li">bootstrap</li>
+            <li class="profesional-Art-li">Ruby</li>
+          </ul>
+          <a href="#3" class="profesional-Art-btm button">See Project</a>
+        </div>
+        <div class="art-container">
+          <h2 class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </h2>
+          <p class="profesional-Art-content">
+            A daily
+            selection of privately personalized reads;
+            no accounts or sign-ups required. has been the
+            industry's standard
+          </p>
+          <ul class="profesional-Art-ul">
+            <li class="profesional-Art-li">html</li>
+            <li class="profesional-Art-li">bootstrap</li>
+            <li class="profesional-Art-li">Ruby</li>
+          </ul>
+          <a href="#4" class="profesional-Art-btm button">See Project</a>
+        </div>
+        <div class="art-container">
+          <h2 class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </h2>
+          <p class="profesional-Art-content">
+            A daily
+            selection of privately personalized reads;
+            no accounts or sign-ups required. has been the
+            industry's standard
+          </p>
+          <ul class="profesional-Art-ul">
+            <li class="profesional-Art-li">html</li>
+            <li class="profesional-Art-li">bootstrap</li>
+            <li class="profesional-Art-li">Ruby</li>
+          </ul>
+          <a href="#5" class="profesional-Art-btm button">See Project</a>
+        </div>
+        <div class="art-container">
+          <p class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </p>
+          <p class="profesional-Art-content">
+            A daily
+            selection of privately personalized reads;
+            no accounts or sign-ups required. has been the
+            industry's standard
+          </p>
+          <ul class="profesional-Art-ul">
+            <li class="profesional-Art-li">html</li>
+            <li class="profesional-Art-li">bootstrap</li>
+            <li class="profesional-Art-li">Ruby</li>
+          </ul>
+          <a href="#6" class="profesional-Art-btm button">See Project</a>
+        </div>
+        <div class="art-container">
+          <p class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </p>
```

Comment:
- [ ] As mentioned by the previous reviewer please use the `p` tag to the `h2` tag it will be better and semantically correct for accessibility as mentioned by the previous reviewer. :point_up: but still it's not changed so kindly fix this issue. :+1: